content/zh/docs/multicluster-management/enable-multicluster/direct-connection.md

```diff
@@ -177,7 +177,7 @@ multicluster:
 kubectl logs -n kubesphere-system $(kubectl get pod -n kubesphere-system -l app=ks-install -o jsonpath='{.items[0].metadata.name}') -f
 ```
 
-## 导入
+## 导入member集群
```

Comment:
`导入member集群` -> `导入成员集群`

Reply:
This is good

Reply:
> This is good

That is true. However, the style of KubeSphere documentation uses `成员集群` as the Chinese translation of `member cluster`. Since your PR is contributing to a document in Chinese, it would be better if you translate the word `member` here.